### ISSUE TITLE
Unable to open Nautilus-based zim files (with PDFs)

### ISSUE THREAD

**Issue by Popolechien** [kiwix/kiwix-android#2788]
There are several files that have been generated via nautilus and consist of PDF files (see, e.g. [this](https://library.kiwix.org/zimgit-post-disaster_en_2022-01) or [that](https://library.kiwix.org/mesptitesquestions_fr_2022-01)).

When trying to open these files I always get an error message "unable to open". Seems a regression to me as it used to work (or even did when I opened [this ticket](../../kiwix-desktop/issues/756))

**Comment by Archenoth** [comment]
Issue #2709 is related.

The tl;dr in there is that it seems as though internal media [gets content URLs that are "https://kiwix.app/"](https://github.com/kiwix/kiwix-android/blob/develop/core/src/main/java/org/kiwix/kiwixmobile/core/reader/ZimFileReader.kt#L248) followed by the internal filepath. External readers try to resolve this as an actual hostname and fail.

It seems commits 3a2ac5f064f7daa7bea6d9707ceb26c55293acf0 (which removed the external-friendly URIs) and 883641076ed5826cad0ef467a883c6ca1400c891 (which updated internal files to use the above hardcoded content prefix) might be the source of this discrepancy.
